_extensions/include-code-files/include-code-files.lua

```diff
@@ -44,20 +109,22 @@ local function transclude (cb)
           end
         end
         number = number + 1
-      end 
+      end
+
       fh:close()
-    end     
+    end
+
     -- remove key-value pair for used keys
     cb.attributes.include = nil
     cb.attributes.startLine = nil
     cb.attributes.endLine = nil
     cb.attributes.dedent = nil
+
     -- return final code block
     return pandoc.CodeBlock(content, cb.attr)
   end
 end
 
 return {
-  { CodeBlock = transclude }
+  { CodeBlock = transclude },
```

Comment:
Just curious, why the comma is better to have here ?

Reply:
This change was made by ``stylua``, which I have set up with [conform.nvim](https://github.com/stevearc/conform.nvim). It would appear that this is just a preferred style guideline in ``lua`` - I'm not too knowledgeable about ``lua``, but this often done in ``python`` fixers (e.g. ``black``) when spreading comma separated lists over many lines.

Reply:
> This change was made by stylua

Good to know ! I don't know exactly which styler we use in Quarto. I'll compare. Let's leave it like this